.travis.yml

```diff
@@ -70,10 +57,5 @@ deploy:
     skip_cleanup: true
     on:
       branch: demo2
-    script: env ROUTE=$SITE_DEMO2_DOMAIN ./.travis/cf-push-2.sh
-  - provider: script
-    skip_cleanup: true
-    on:
-      branch: antora
-    script: env ROUTE=$SITE_ANTORA_DOMAIN ./.travis/cf-push-2.sh
+    script: ./.travis/cf-push-2.sh
```

Comment:
Shouldn't we get rid of `cf-push-2.sh` and just make `.travis/cf-push.sh` take a param for the Bluemix org?
`script: ./.travis/cf-push.sh BLUEMIX_ORGANIZATION` (default to BLUEMIX_ORGANIZATION if nothing passed in)
`script: ./.travis/cf-push.sh BLUEMIX_ORGANIZATION_2`